package.json

```diff
@@ -416,6 +414,19 @@
           ],
           "default": "Workspace",
           "description": "When the setting that hides the option to choose where to save new connections is checked, this setting sets if and where new connections are saved."
+        },
+        "mdb.playgroundTemplate": {
+          "type": "string",
+          "enum": [
```

Comment:
Are we thinking about adding more templates in the future? Otherwise maybe this can just be a boolean

Reply:
I was thinking we may want more in the future but to be honest can't really think of a 3rd option now. I will change it to boolean.